eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/ImmutableList.java

```diff
@@ -98,6 +99,24 @@
         return this.collect(each -> function.valueOf(each, index[0]++));
     }
 
+    /**
+     * @since 11.0
```

Comment:
Missing Javadoc.  Seems like it is missing from the rest of the files too but I am not adding individual comments :)

Reply:
Good point. I will write some javadoc for the interface methods and overrides.

Reply:
Added Javadoc for several interfaces and one utility class @prathasirisha. Thanks for pointing out.